modules/system/console/asset/NpmCommand.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace System\Console\Asset;
```

Comment:
This should probably be in the `System\Console\Asset\Npm` namespace shouldn't it?

Reply:
@jaxwilko any thoughts on the above?

Reply:
I kinda prefer it to be above the commands as it's an abstract but no real preference other than that

Reply:
Sure it's an abstract, but it's an abstract that will only ever be used by the classes in the npm folder, so I think it makes sense for it to live there instead.

Reply:
Fair, it shall be moved :)